Cargo.toml

```diff
@@ -14,29 +14,56 @@ edition = "2018"
 [features]
 default = ["connect"]
 connect = ["stream", "tokio/net"]
-tls = ["native-tls", "tokio-native-tls", "stream", "tungstenite/tls"]
+native-tls = ["native-tls-crate", "tokio-native-tls", "stream", "tungstenite/native-tls"]
+native-tls-vendored = ["native-tls", "native-tls-crate/vendored", "tungstenite/native-tls-vendored"]
+rustls-tls = ["rustls", "tokio-rustls", "stream", "tungstenite/rustls-tls", "webpki", "webpki-roots"]
 stream = []
 
 [dependencies]
 log = "0.4"
 futures-util = { version = "0.3", default-features = false, features = ["async-await", "sink", "std"] }
 pin-project = "1.0"
-tokio = { version = "1.0.0", default-features = false, features = ["io-util"] }
+tokio = { version = "1.2.0", default-features = false, features = ["io-util"] }
```

Comment:
Oh sorry, that's my habit of updating versions. This works with 1.0.0 so I'll switch back the version number.
But as libraries don't have a lock file, how do we ensure that no new features from say 1.2.0 are currently used?

If the CI builds (or any other user) then cargo will create a Cargo.lock file with the latest version instead of 1.0.0. Somebody would need to change the version to `=1.0.0` from time to time (locally only) to see whether everything still works.

Reply:
> But as libraries don't have a lock file, how do we ensure that no new features from say 1.2.0 are currently used?

This is a very valid concern. I don't really know what the best practices are here, or if some even exist regarding this matter. It is also possible that nobody actually uses compatible-but-older versions of libraries. :)

My comment was mainly in the terms of "a PR should only contain changes the connected with achieving the stated goal" to avoid scope creep.

Reply:
That is very true. I will revert the other version upgrades as well then 🙇‍♂️

Reply:
> I will revert the other version upgrades as well then

It seems that these changes were not yet pushed?

Reply:
Yeah. With the latest change I was just focused on trying to find a nice solution for the feature conflicts. I'll add this together with the changelog additions soon.